tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export KOPS_FEATURE_FLAGS="SpecOverrideFlag,"
```

Comment:
I don't think I want this flag to get back in.

Reply:
I have removed it in https://github.com/kubernetes/kops/pull/15144/commits/5b95d3073aacbb80d14c3be7a9958cce2043cd1b